### PR TITLE
revamp: auth ui

### PR DESCRIPTION
**Pull request by Abhay-2811**
For: #23 

Done: UI change, passwordless email auth

Note: @Nasfame  I have updated and checked the client id's I'll DM them to you

### CODE REVIEW

**Comment by Abhay-2811** [comment]
No merge conflicts 💯

File: src/app/explore/ethRPC.ts
Comment (Nasfame) [inline src/app/explore/ethRPC.ts:1]: Why do we have this file, we already discussed having this file in utils with the name: utils/web3auth.ts

File: src/app/explore/page.tsx
Comment (Nasfame) [inline src/app/explore/page.tsx:1]: @techyNonso kindly review this change

File: next.config.js
Comment (Nasfame) [inline next.config.js:1]: LGTM

**Review by Nasfame — CHANGES_REQUESTED** [review]

**Review by techyNonso — COMMENTED** [review]
What is saphire_devnet please

**Comment by Abhay-2811** [comment]
Web3 testnet, they have 2 testnets one is named sapphire

**Review by techyNonso — COMMENTED** [review]
LGTM

File: src/app/explore/ethRPC.ts
Comment (Abhay-2811) [reply src/app/explore/ethRPC.ts:1]: oh i pulled and i got changed

Comment (Abhay-2811) [reply src/app/explore/ethRPC.ts:1]: done

**Review by Nasfame — APPROVED** [review]
LGTM, untested

**Comment by Nasfame** [comment]
Fix #23

**Comment by Nasfame** [comment]
Resolved #23

**Pull request merged**